docker/sysctl-setter/Dockerfile

```diff
@@ -12,6 +12,14 @@ while true; do\n\
   if [ "$SVAL" -ne 1 ]; then\n\
     sysctl -w vm.swappiness=1\n\
   fi\n\
+  FVAL=$(sysctl -n fs.file-max)\n\
```

Comment:
From my googling so far, it seems like most recent kernels set this to an insanely high value anyway. Can we leave this out?

Reply:
It doesn't hurt if we do `le`